src/ign_TEST.cc

```diff
@@ -62,6 +62,10 @@ TEST(check, SDF)
   std::string pathBase = PROJECT_SOURCE_PATH;
   pathBase += "/test/sdf";
 
+  std::string datapath;
+  sdf::testing::TestDataPath(datapath);
+  std::string g_ignCommand = datapath + "/../ign_tools/ign";
```

Comment:
I believe the `g_` prefix was signaling that it was a global variable.

Reply:
I reverted all changes to this file, as we aren't currently building this test in bazel (because of ign-tools dependency)